docs/pages/all.html

```diff
@@ -8,7 +8,7 @@
 
 <p>Welcome to the alpha site for the Draft U.S. Web Design Standards project.</p>
 
-<p>This Draft Web Design Standards is intended to provide reference for designers, developers, as well as anyone else interested in the design and frontend principles that make up the building blocks of federal government websites.</p>
+<p>The Draft U.S. Web Design Standards is intended to provide reference for designers, developers, as well as anyone else interested in the design and frontend principles that make up the building blocks of federal government websites.</p>
```

Comment:
I think we agreed earlier that we'd use it as a plural, since it sounds more natural.